server/models/index.ts

```diff
@@ -93,6 +95,11 @@ const models = {
 /**
  * Relationships
  */
+
+// AccountingCategory
+models.AccountingCategory.belongsTo(models.Collective, { foreignKey: 'CollectiveId', as: 'collective' });
+models.AccountingCategory.hasMany(models.Expense, { foreignKey: 'AccountingCategoryId', as: 'expenses' });
```

Comment:
In order to have `Expenses.getAccountingCategory()` method defined, we need to add
```ts
models.Expense.belongsTo(models.AccountingCategory, { foreignKey: 'AccountingCategoryId', as: 'accountingCategory' });
```

Reply:
I'm not using that anymore, will remove it from the model